docs/user-guide/concepts.md

```diff
@@ -505,6 +500,40 @@ can either query such objects directly, or via `kubectl hns describe NAMESPACE`.
 The event will include machine-readable and human-readable information about the
 problem, and will generally require human intervention to resolve.
 
+<a name="admin-managed-labels"/>
+
+### Managed labels and annotations
+
+***Managed labels and annotations are planned for HNC v1.0+***
+
+Just as certain objects can be propagated from parent namespaces to their
+descendants, so can certain labels and annotations on namespaces. For example,
+an admin may define an `environment:prod` label on a parent namespace, and
```

Comment:
Suggestion: Use a prefixed label in this example. K8s recommends prefixed labels, ref. https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set

Reply:
Done, thanks